src/styles/base/_tables.scss

```diff
@@ -0,0 +1,30 @@
+table {
+
+	border-collapse: collapse;
+	width: 100%;
+	margin: 0 0 #{ .75 * $base-line-height };
+
+	th,
```

Comment:
These don't need to be nested, the unprefixed selector is specific enough as the table inferred.